test/test_ops.py

```diff
@@ -972,10 +972,10 @@ def test_conj_view(self, device, dtype, op):
     def test_neg_view(self, device, dtype, op):
         if not op.test_neg_view:
             self.skipTest("Operation not tested with tensors with negative bit.")
-        math_op_physical = torch.neg
+        math_op_physical = torch.clone
```

Comment:
can you leave a comment here explaining why we have clone here instead of neg (which is expected given the name of the var `math_op_physical`)

Reply:
inserting the PR description here would be sufficient too